extensions/merge-conflict/src/mergeConflictParser.ts

```diff
@@ -49,7 +51,11 @@ export class MergeConflictParser {
 				}
 
 				// Create a new conflict starting at this line
-				currentConflict = { startHeader: line };
+				currentConflict = { startHeader: line, commonAncestors: null };
+			}
+			// Are we within a conflict block and is this a common ancestors marker? |||||||
+			else if (currentConflict && line.text.startsWith(commonAncestorsMarker)) {
+				currentConflict.commonAncestors = line;
```

Comment:
- [x] Incorporated the first solution for both markers, now we are using just the first token of each type, just like TypeScript does.